core/src/banking_stage/immutable_deserialized_packet.rs

```diff
@@ -54,21 +52,21 @@ impl ImmutableDeserializedPacket {
         let is_simple_vote = packet.meta().is_simple_vote_tx();
 
         // drop transaction if prioritization fails.
-        let mut priority_details = sanitized_transaction
-            .get_transaction_priority_details(packet.meta().round_compute_unit_price())
+        let mut compute_budget_details = sanitized_transaction
+            .get_compute_budget_details(packet.meta().round_compute_unit_price())
             .ok_or(DeserializedPacketError::PrioritizationFailure)?;
 
         // set priority to zero for vote transactions
```

Comment:
nit: `// set compute_unit_price to zero for vote transactions`

Reply:
e8d6e6ea7771e373415be3da0e2453fab1d0d4d1